contract/scripts/propose-start-contract.sh

```diff
@@ -20,3 +20,16 @@ agd tx gov submit-proposal swingset-core-eval "$PERMIT" "$SCRIPT" \
 set +x # not so noisy for this part
 . /usr/src/upgrade-test-scripts/env_setup.sh
 voteLatestProposalAndWait
+
+# Give the core-eval script a block or two to run.
+waitForBlock 2
+
+parseInstances() {
+  # extract latest value; parse smallCaps; yield 1 line per entry
+  jq -c '.value | fromjson | .values[-1] | fromjson | .body[1:] | fromjson | .[]'
+}
+
+# check that the contract was actually started
```

Comment:
Hard for me to track this down right now, but will a failure of this check result in the `yarn start:contract` to fail? I don't see any `exit` here if that were the case, but I also don't know of the error code would bubble all the way back.

Reply:
yes, the error code does bubble all the way back

Reply:
Oh right, if grep doesn't find a match, it's a pipe fail. No need to explicitly exit.